src/pages/data_submission/v2/v2-models.tsx

```diff
@@ -0,0 +1,342 @@
+  }
+}
+
+export class NihAnvilUse extends StudyProperty {
+  static readonly key = 'nihAnvilUse'
+  static readonly YES_NHGRI_YES_PHS_ID = 'I am NHGRI funded and I have a dbGaP PHS ID already'
+  static readonly YES_NHGRI_NO_PHS_ID = 'I am NHGRI funded and I do not have a dbGaP PHS ID'
+  static readonly NO_NHGRI_YES_ANVIL = 'I am not NHGRI funded but I am seeking to submit data to AnVIL'
+  static readonly NO_NHGRI_NO_ANVIL = 'I am not NHGRI funded and do not plan to store data in AnVIL'
+  static readonly NIH_ANVIL_USE_RADIOGROUP_OPTIONS = [
+    { text: NihAnvilUse.YES_NHGRI_YES_PHS_ID, name: NihAnvilUse.YES_NHGRI_YES_PHS_ID },
+    { text: NihAnvilUse.YES_NHGRI_NO_PHS_ID, name: NihAnvilUse.YES_NHGRI_NO_PHS_ID },
+    { text: NihAnvilUse.NO_NHGRI_YES_ANVIL, name: NihAnvilUse.NO_NHGRI_YES_ANVIL },
+    { text: NihAnvilUse.NO_NHGRI_NO_ANVIL, name: NihAnvilUse.NO_NHGRI_NO_ANVIL },
+  ]
+
+  constructor(value?: string, studyId?: number, studyPropertyId?: number) {
+    super(NihAnvilUse.key, 'String' as StudyPropertyType, value, studyId, studyPropertyId)
+  }
+
+  static requiresNIHAdministrativeInformation(value: string | null | undefined): boolean {
+    if (!value) {
+      return false
+    }
+    return [NihAnvilUse.YES_NHGRI_YES_PHS_ID, NihAnvilUse.YES_NHGRI_NO_PHS_ID, NihAnvilUse.NO_NHGRI_YES_ANVIL].includes(value)
+  }
+}
+
+export class NihGenomicProgramAdministratorName extends StringStudyProperty {
+  static readonly key = 'nihGenomicProgramAdministratorName'
+  constructor(value?: string, studyId?: number, studyPropertyId?: number) {
+    super(NihGenomicProgramAdministratorName.key, 'NIH Genomic Program Administrator Name', 'Enter the NIH Genomic Program Administrator\'s name ', value, studyId, studyPropertyId)
+  }
+}
+
+export class CollaboratingSites extends StudyProperty {
+  static readonly key = 'collaboratingSites'
+  constructor(value: string[], studyId?: number, studyPropertyId?: number) {
+    super(CollaboratingSites.key, 'Json' as StudyPropertyType, value, studyId, studyPropertyId)
+  }
+}
+
+export class ControlledAccessRequiredForGenomicSummaryResultsGSRRequiredExplanation extends StringStudyProperty {
+  static readonly key = 'controlledAccessRequiredForGenomicSummaryResultsGSRRequiredExplanation'
+  constructor(value?: string, studyId?: number, studyPropertyId?: number) {
+    super(ControlledAccessRequiredForGenomicSummaryResultsGSRRequiredExplanation.key, 'If yes, explain why controlled access is needed for GSR.', 'Enter explanation here', value, studyId, studyPropertyId)
+  }
+}
+
+export interface Study {
+  studyId?: number
+  uuid?: string
+  name?: string
+  description?: string
+  dataTypes?: string[]
+  piName?: string
+  publicVisibility?: boolean
+  datasetIds?: number[]
+  datasets?: Dataset[]
+  properties?: StudyProperty[]
+  alternativeDataSharingPlan?: FileStorageObject
+  createDate?: string // Date?
```

Comment:
I'd really like to try and standardize on this question, we currently have an all-of-the-above strategy.

<img width="451" height="495" alt="Image" src="https://github.com/user-attachments/assets/0b929473-4d95-4124-b298-94895abf4b94" />

Reply:
Maybe at a future point we can have a DateInterface that allows number|string|Date but has methods like `toDate`, ` toString`,  or `toTimestamp`?